sql/src/main/java/io/crate/cluster/gracefulstop/DecommissioningService.java

```diff
@@ -195,10 +195,23 @@ private void removeRemovedNodes(ClusterChangedEvent event) {
     protected void doStart() {
         // add listener here to avoid guice proxy errors if the ClusterService could not be build
         clusterService.addListener(this);
+        if (GRACEFUL_STOP_REALLOCATE_SETTING.setting().get(settings).booleanValue()
```

Comment:
We can add a `Setting.Property.Deprecated` property to the `GRACEFUL_STOP_REALLOCATE_SETTING` definition and won't have to do the deprecation logging manually here.